src/ESPiLight.cpp

```diff
@@ -156,8 +156,8 @@ ESPiLight::ESPiLight(int8_t outputPin) {
   _rawCallback = nullptr;
 
   if (_outputPin >= 0) {
-    pinMode(_outputPin, OUTPUT);
-    digitalWrite(_outputPin, LOW);
+    pinMode(static_cast<uint8_t>(_outputPin), OUTPUT);
```

Comment:
Why `static_cast<uint8_t>` and not `(uint8_t)`?

Reply:
Its just a matter of personal taste at this point. I normally use static_cast everywhere as its checked by the compiler and guaranteed to be castable.